features/git-rename-branch/non_feature_branch/without_remote_origin/without_open_changes.feature

```diff
@@ -0,0 +1,36 @@
+Feature: git rename-branch: renaming a non-feature branch without a remote repo (without open changes)
+
+  (see ../with_remote_origin/with_tracking_branch/with_open_changes.feature)
+
+
+  Background:
+    Given my repo does not have a remote origin
+    And I have local branches named "qa" and "production"
+    And my non-feature branches are configured as "qa" and "production"
+    And the following commits exist in my repository
+      | BRANCH     | LOCATION | MESSAGE           |
+      | main       | local    | main commit       |
+      | production | local    | production commit |
+      | qa         | local    | qa commit         |
+    And I am on the "production" branch
+
+
+  Scenario: error when trying to rename
+    When I run `git rename-branch production renamed-production`
+    Then it runs no Git commands
+    And I get the error "The branch 'production' is not a feature branch. Run 'git rename-branch production renamed-production -f' to force the rename, then reconfigure git-town on any other clones of this repo."
```

Comment:
What exactly do you want to break up here? The error message string? I think we should do that. It is important to see both error messages (1. the branch is not a feature branch, 2. guidance on how to rename it anyways), but the spec shouldn't care in which order they happen.

```cucumber
And I get the error "The branch 'production' is not a feature branch."
And I get the error "Run 'git rename-branch production renamed-production -f' to force the rename, then reconfigure git-town on any other clones of this repo."